src/NuGet.CommandLine/Commands/ListCommand.cs

```diff
@@ -114,22 +162,17 @@ public ICollection<string> Source
                      ***********************************************/
                     foreach (var p in packages)
                     {
-                        Console.PrintJustified(0, p.Identity.Id);
-                        if (!AllVersions)
-                        {
-                            Console.PrintJustified(1, p.Identity.Version.ToNormalizedString());
-                            Console.PrintJustified(1, p.Description);
-                        }
-                        else
+                        Console.PrintJustified(0, p.Id);
+                        Console.PrintJustified(1, p.Version.ToString());
+                        Console.PrintJustified(1, p.Description);
+                        if (p.LicenseUrl != null && !string.IsNullOrEmpty(p.LicenseUrl.OriginalString))
```

Comment:
this could be simplified:
if (!string.IsNullOrEmpty(p.LicenseUrl?.OriginalString))